cfgov/v1/views/reports.py

```diff
@@ -17,6 +19,35 @@ def process_tags(queryset):
     return ", ".join([tag for tag in queryset])
 
 
+def process_docket_numbers(queryset):
+    """Grab the one to many Docket Numbers values"""
+    return ",".join([d.docket_number for d in queryset])
+
+
+def process_content(page_content):
+    content = ""
+    soup = BeautifulSoup(str(page_content), "html.parser")
+    para = soup.findAll(["p", "h5"])
+    for p in para:
+        content += p.get_text()
+        link = p.find("a", href=True)
+        if link:
+            content += ": "
+            content += link["href"]
+        content += "\n"
+    return content
+
+
+def process_products(queryset):
+    """Grab the one to many Docket Numbers values"""
+    return ", ".join([p.product for p in queryset])
```

Comment:
Same as with `process_docket_numbers `, the report can use the `product_strings` property instead.